libraries/app/include/graphene/app/api.hpp

```diff
@@ -185,11 +185,11 @@ namespace graphene { namespace app {
                                                                         unsigned limit = 100,
                                                                         uint32_t start = 0) const;
          /**
-          * @brief Get list of orders with b:a pair
+          * @brief Get asset pairs with b:a pair
```

Comment:
"asset pairs" doesn't make sense. Please talk to traders to see what should be put here.

Reply:
OK. Sure. Thanks !